website/docs/cloud-docs/api-docs/projects.mdx

```diff
@@ -557,6 +557,102 @@ $ curl \
 }
 ```
 
+## Add/update tag-bindings on a workspace
+
+This endpoint adds keys and values or updates values of tag-bindings on an existing resource by key.
+It does not remove any keys from the collection. This endpoint is useful when you want to ensure a
```

Comment:
Consider:

```suggestion
It can not be used to remove any tag bindings from the resource. This endpoint is useful when you want to ensure a tag binding
```